configs/restorers/basicvsr/basicvsr_vimeo90k_bd.py

```diff
@@ -16,7 +16,7 @@
 
 # dataset settings
 train_dataset_type = 'SRVimeo90KMultipleGTDataset'
-val_dataset_type = 'SRTestMultipleGTDataset'
+val_dataset_type = 'SRCustomMultipleGTDataset'
```

Comment:
What is `Custom`

Reply:
Will change to `SRFolderMultipleGTDataset`, maching `SRFolderDataset`.

Reply:
Fine